### ISSUE TITLE
A way to remove images?

### ISSUE THREAD

**Issue by tilkinsc** [chubin/wttr.in#297]
I want it to display column 80, which it can be reduced down to ~72 lines without pictures as I calculated.

**Comment by chubin** [comment]
@tilkinsc It is not yet possible, but will be possible soon, after the next major update (so called v2.wttr.in).
Currently you can use the narrow output for that:
```
curl wttr.in/?n
```

**Comment by tilkinsc** [comment]
Noice. Do you want me to close?

**Comment by chubin** [comment]
@tilkinsc No, please do not. Let's close it as soon as the feature is implemented
